### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) combines a series of small changes PRs: #639, #640, #641, #642, #643, and #644.  

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/getting-started.html
https://docs.magento.com/user-guide/getting-started/login-credentials.html
https://docs.magento.com/user-guide/magento/magento-account-share.html
https://docs.magento.com/user-guide/quick-tour/engage.html
https://docs.magento.com/user-guide/quick-tour/self-service-tools.html
https://docs.magento.com/user-guide/quick-tour/market-opportunities.html

### CODE REVIEW

File: src/getting-started/login-credentials.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The address for your store Admin is set up during the installation. The default
 
 ## Magento Account
 
-Your Magento account provides access to information about your products and services, account settings, billing history, and support resources. to access your account, visit the Magento site and click the My Account link in the header.
+Your Magento account provides access to information about your products and services, account settings, billing history, and support resources. To access your account, visit the Magento site and click the My Account link in the header.
```
Comment (hickskenh) [inline src/getting-started/login-credentials.md:29]: visit or go to?

File: src/magento/magento-account-share.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ The following instructions are written from the perspective of a shared user who
    ![]({% link images/images/magento-account-shared-support.png %}){: .zoom}
    _Support_
 
-   You can use the the Magento Help Center from the shared account to search for articles and troubleshooting information, find patches for known issues, and create support tickets.
+   You can use the Magento Help Center from the shared account to search for articles and troubleshooting information, find patches for known issues, and create support tickets.
```
Comment (hickskenh) [inline src/magento/magento-account-share.md:83]: link Magento Help Center?

File: src/getting-started/login-credentials.md (hunk 1 above)
Comment (leslietilling) [reply src/getting-started/login-credentials.md:29]: I think visit is OK for this type of thing.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**